octo-core/lib/octocore/mailer/subscriber_mailer.rb

```diff
@@ -0,0 +1,32 @@
+require 'octocore'
+
+module Octo
+  module Mailer
+    class SubscriberMailer
+      @queue = :subscriber_notifier
+
+      # Method for the scheduler to call
+      # Counts the number of subscriber in the last 24 hours
+      #  and then sends a mail with subscriber count to the
+      #  email mentioned
+
+      def perform (from=nil)
+        if from.nil?
+          subscribers = Octo::Subscriber.where(created_at: 24.hours.ago..Time.now.floor)
+        else
+          subscribers = Octo::Subscriber.where(created_at: from..Time.now.floor)
+        end
```

Comment:
Please uncomment this. The functionality should be present in the code. You could add a check on top level `Octo::Email.send` to send email only if the required keys are present. Remaining part of the code should be abstracted out of this thing. This goes in line with the require 'email' comment.